consensus/wal.go

```diff
@@ -72,6 +76,9 @@ type baseWAL struct {
 	group *auto.Group
 
 	enc *WALEncoder
+
+	flushTicker   *time.Ticker
+	flushInterval time.Duration
 }
 
 func NewWAL(walFile string, groupOptions ...func(*auto.Group)) (*baseWAL, error) {
```

Comment:
We should add a godoc comment saying that we sync wal every 2s. Otherwise it's not possible to know it's doing that